tsdfileapi/api.py

```diff
@@ -2562,10 +2560,13 @@ def get(self, tenant: str, table_name: str = None) -> None:
                     for row in self.db.table_select(table_name, query, exclude_endswith = ["_audit", "_metadata"]):
                         if not first:
                             self.write(',')
+                        else:
+                            self.write('[')
```

Comment:
There are currently two `set_status(200)` _ one should be enough. Maybe put it before the `self.write('[')`? At this point the body is written to the response and therefore the header, containing the status code, is already sent out (I assume this is what the `flush()` is doing)